mathesar_ui/src/systems/users-and-permissions/UserDetailsForm.svelte

```diff
@@ -140,6 +152,14 @@
     />
   {/if}
 
+  <UserFormInput
+    label="Preferred Language *"
```

Comment:
Also "Preferred Language" needs to be changed to "Display Language" here.

Reply:
0e8e674